src/stores/store.jsx

```diff
@@ -991,6 +991,7 @@ class Store {
           invest: 'invest',
           redeem: 'redeem',
           price_id: 'ethereum',
+          pureEthereum: true,
```

Comment:
Earn v1 one of the "disabled" vaults were throwing an error. I doubt many people have funds in the asset anymore, But I did, so noticed it and fixed it.